src/main/java/org/kainos/ea/DropwizardApplicationApplication.java

```diff
@@ -3,6 +3,9 @@
 import io.dropwizard.Application;
 import io.dropwizard.setup.Bootstrap;
 import io.dropwizard.setup.Environment;
+import io.federecio.dropwizard.swagger.SwaggerBundle;
+import io.federecio.dropwizard.swagger.SwaggerBundleConfiguration;
+import org.kainos.ea.resources.JobspecController;
 import org.kainos.ea.resources.TestController;
 
 public class DropwizardApplicationApplication extends Application<DropwizardApplicationConfiguration> {
```

Comment:
You didn't add this but `DropwizardApplicationApplication`?

Reply:
Think I must have added extra Application in name section of the Setup